src/Sylius/Bundle/ApiBundle/Controller/GetSalesStatisticsAction.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Controller;
+
+use Sylius\Bundle\ApiBundle\Query\Admin\GetSalesStatistics;
+use Sylius\Component\Core\Sales\ValueObject\SalesPeriod;
```

Comment:
```suggestion
use Sylius\Component\Core\Statistics\ValueObject\Period;
```
```suggestion
use Sylius\Component\Core\Statistics\Period;
```

From BC point of view, it would be better to put this into
`ApiBundle\Statistics\Period` and once feature will mature, put into `Core`

Reply:
As far as I know, we will remove the `@experimental` in Sylius 2.0, and the ApiBundle will also become part of the BC Policy